app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -434,41 +433,59 @@ class DefaultToolbarMenu(
             onItemTapped.invoke(ToolbarMenu.Item.RequestDesktop(checked))
         }
 
-        val customizeReaderView = SimpleBrowserMenuItem(
+        val customizeReaderView = BrowserMenuImageText(
             label = context.getString(R.string.browser_menu_customize_reader_view),
-            textColorResource = primaryTextColor()
+            imageResource = R.drawable.ic_readermode_appearance,
+            iconTintColorResource = primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.CustomizeReaderView)
         }
 
+        val openInApp = BrowserMenuHighlightableItem(
```

Comment:
Similar nit as above: Why are we adding this here? Seems related to a different PR.

Reply:
It was taken out as a part of the original design, and the updated designs add it back in. I could open a separate PR but that seemed wasteful

Reply:
PRs are cheap. 🙂 

It's a good rule of thumb to keep isolated work separately so there is context when someone needs to debug why a change was added to a separate issue. Smaller PRs are also easier for the reviewer.

How about separating this into a different commit so it doesn't fall under "Remove disabled color for icons"?

Reply:
Sure, I'll tag you ;)